### ISSUE TITLE
Poor performance of Bottom Sheet on mobile

### ISSUE THREAD

**Issue by Codebreaker101** [quasarframework/quasar#8341]
**Describe the bug**
I'm opening a new issue since the previous one was closed (#8006). Testing on the latest release with empty project and the lag persists. Basically, when opening BottomSheet there is noticeable stuttering compared to a dialog opening transition. Closing transition is smooth. 

**Codepen/jsFiddle/Codesandbox (required)**
https://quasar.dev/quasar-plugins/bottom-sheet#Usage

**To Reproduce**
1. Initiate blank quasar project with everything staying default
2. Add capacitor support
3. Copy example of the [bottom-sheet documentation](https://quasar.dev/quasar-plugins/bottom-sheet#Usage) into `src/pages/Index.vue`
4. Build android app `quasar build -m capacitor -T android`
5. Install on mobile and open

**Expected behavior**
For the animation to be as smooth like opening a dialog.

**Screenshots**
Demonstration video of the lag: https://youtu.be/j4AtB65e0UQ

**Platform (please complete the following information):**
OS: Android
Node: v12.18.4 + version that quasar.dev was compiled with.
NPM: 6.14.7
Yarn: 1.22.5
Browsers: Firefox Android 82.1.1, Opera Android 60.3.3004.55692, Edge Android 45.09.4.5083, Brave Android 1.16.74, Chrome Android 86.0.424.185, Brave Desktop 1.12.105 (84.0.4147.105)
iOS: N/A
Android: 8
Electron: 9.2.1

_package.json_
```
"dependencies": {
    "@quasar/extras": "^1.0.0",
    "core-js": "^3.6.5",
    "quasar": "^1.0.0"
  },
  "devDependencies": {
    "@quasar/app": "^2.0.0",
    "babel-eslint": "^10.0.1",
    "eslint": "^6.8.0",
    "eslint-config-standard": "^14.1.0",
    "eslint-loader": "^3.0.3",
    "eslint-plugin-import": "^2.14.0",
    "eslint-plugin-node": "^11.0.0",
    "eslint-plugin-promise": "^4.0.1",
    "eslint-plugin-standard": "^4.0.0",
    "eslint-plugin-vue": "^6.1.2"
  },
```

**Comment by ibrainventures** [comment]
If youre issue only happens on Android, the problem is issued by the border radius 
which is only top left and top right.

if the border-radius is **on all edges**  **or** there is **no border-radius** it should fly fluent up.

the vendor / quasar css behind is:

```
.q-dialog__inner > div {
...
	border-top-left-radius: 4px;
        border-top-right-radius: 4px;
...
}

.q-card {
..
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
..
}
```

My solution was to fly in with all edges border-radius and after a setTimeout i am setting the bottom borders
to no radius .. so it looks not to strange if it has no borders on top during fly-up.
   
```
   Inside Data:

   styleDummy: {},
   styleObj: { 'border-bottom-left-radius': '0px !important', 'border-bottom-right-radius': '0px !important' },

   mounted

    setTimeout(() => {
      this.styleDummy = this.styleObj
    }, 400)

  <q-card
    class
    :style="styleDummy"
```

May not the finest solution, but it makes the flyin smooth.

**Comment by Codebreaker101** [comment]
@ibrainventures That is an awesome find! Thank you very much!

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.0.0-rc.1.
Will try to port it to v1 as well, but later.
